src/components/Input.react.js

```diff
@@ -162,17 +165,26 @@ Input.propTypes = {
     /**
      * This attribute indicates whether the value of the control can be automatically completed by the browser.
      */
-    autocomplete: PropTypes.string,
+    autocomplete: PropTypes.oneOfType([
+        PropTypes.string,
+        PropTypes.bool
```

Comment:
also the actual accepted type per spec is string, if we wanna accept boolean on Python/R side, they need to be coerced/casted to `on` and `off`

Reply:
Ah ok. But these properties https://github.com/iandevlin/html-attributes/blob/master/boolean-attributes.json should accept True/False right?